main-site/components/events-calender-section/EventsCalenderSection.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import EventsGrass from '../../../shared-ui/images/events-grass.svg';
+import EventsCactus from '../../../shared-ui/images/events-cactus.svg';
+import {
+  StyledEventsCactus,
+  StyledEventsGrass,
+  StyledEventsContainer,
+  StyledEventsCalender, 
+  StyledH3
+} from './EventsCalenderSection.styles';
+import { min } from '../../../shared-ui/lib/responsive';
+import useMatchMedia from 'react-use-match-media';
+import DesktopTexts from './DesktopTexts';
+import NonDesktopTexts from './NonDesktopTexts';
+
+const EventsCalenderSection: React.FC = () => {
+  const isDesktop = useMatchMedia(min.tablet);
+  return (
+    <>
```

Comment:
turn this empty tag into a styled div and add more top padding (check each screen size to make sure it's an appropriate amount) 
Right now the section is too close to the previous one - check the figma for the right amount. Also, the past photos is already merged into main so please merge main into your branch + resolve merge conflicts then make sure your section is right after the past photos. Then adjust the padding according to the design.
<img width="1512" alt="Screen Shot 2022-10-09 at 9 26 00 AM" src="https://user-images.githubusercontent.com/59738880/194759451-bdff5b4c-df00-447e-a052-c98d9c9b2d5c.png">